doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,29 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+* LwM2M:
+
+  - Fixed a bug, where FOTA socket was not closed after the download (PULL mode).
+  - Allow to specify how early before the timeout shall Register Update be sent.
```

Comment:
```suggestion
  - Allowed to specify how early before the time-out the Register Update will be sent.
```

Reply:
How is this specified? Is it a Kconfig option, or a parameter to a function?

Reply:
It's a Kconfig option.